benchmark/metrics.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Benchmark
```

Comment:
Did you mean for this to be in Benchmark::Metrics? Otherwise I think it's fine to stuff it under Benchmark in benchmark.rb